Feature/adds new transaction fields (#55)
inloco/incognia-api-java: README.md

```java
IncogniaAPI api = new IncogniaAPI("client-id", "client-secret", Region.BR);
try {
     RegisterLoginRequest registerLoginRequest =
        RegisterLoginRequest.builder()
          .installationId("installation id")
          .accountId("account id")
          .externalId("external id")

>           .evaluateTransaction(true) // can be omitted if you want this to be evaluated

Reviewer: ```suggestion
          .evaluateTransaction(true) // can be omitted as it uses true as the default value
```
What do you think?

I believe that we should also include a section explaining the usage of the eval parameter, as we do on the [Go lib](https://developer.incognia.com/docs/br/v6/apis/feedback/feedback-api#understanding-event-types).

Author: i'll add it

Author: done

                    .build())
            .coordinates(new Coordinates(40.74836007062138, -73.98509720487937))
            .build();
     
     Map<AddressType, Address> addresses = Map.of(
         AddressType.SHIPPING, address
         AddressType.BILLING, address);
     
     List<PaymentMethod> paymentMethods = new ArrayList<>();
        paymentMethods.add(
            PaymentMethod.builder()
                .creditCardInfo(
                    CardInfo.builder()
                        .bin("123456")
                        .expiryMonth("10")
                        .expiryYear("2028")
                        .lastFourDigits("4321")
                        .build())
                .type(CardType.CREDIT_CARD)
                .build());
        
     RegisterPaymentRequest registerPaymentRequest =
         RegisterPaymentRequest.builder()
             .installationId( "installation-id")
             .accountId("account-id")
             .externalId("external-id")
             .addresses(addresses)

>              .evaluateTransaction(true) // can be omitted if you want this to be evaluated

Reviewer: Same as above comment

}
```

### Registering Payment or Login without evaluating its risk assessment

Turning off the risk assessment evaluation allows you to register a new transaction (Login or Payment), but the response (`TransactionAssessment`) will be empty. For instance, if you're using the risk assessment only for some payment transactions, you should still register all the other ones: this will avoid any bias on the risk assessment computation.

To register a login or a payment without evaluating its risk assessment, you should use the `evaluateTransaction` boolean set to false

Example:

```java
RegisterLoginRequest registerLoginRequest =
        RegisterLoginRequest.builder()
          .installationId("installation id")
          .accountId("account id")
          .externalId("external id")
          .evaluateTransaction(false)
          .build();
```

> Would return an empty risk assessment response:
> 
> ``{}``

Reviewer: I believe this could be removed, as line 190 already mentions that the response will be empty
```suggestion

```
